testdata/p4_16_samples/invalid-hdr-warnings3-bmv2.stf

```diff
@@ -0,0 +1,16 @@
+# STF test for /home/vpeschax/networking.switching.barefoot.bf-p4c-tools/compilers/p4c/build/testgen/testgen-p4c-bmv2/invalid-hdr-warnings3.out/invalid-hdr-warnings3
```

Comment:
I would remove these hardcoded paths and the comments are not quite necessary yet.